providers/aws/services/securityhub/securityhub_service.py

```diff
@@ -0,0 +1,91 @@
+import threading
+from dataclasses import dataclass
+
+from lib.logger import logger
+from providers.aws.aws_provider import generate_regional_clients
+
+
+################## SECURITYHUB
+class SECURITYHUB:
```

Comment:
```suggestion
class SecurityHub:
```

Reply:
Python naming convention, please change the references too.

Reply:
Done